brianlong/www.validators.app
# 184633886 csv for api endpoints

#### What's this PR do?
- enable csv option for api endpoints

#### How should this be manually tested?
- check every API endpoint by adding ".csv" at the end of the path
- check if standard API queries work as usual
- check if all front pages utilizing API are working properly
- run tests

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/184633886)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
